packages/core/upload/server/bootstrap.js

```diff
@@ -37,7 +37,8 @@ module.exports = async ({ strapi }) => {
 
   await registerPermissionActions();
 
-  await getService('metrics').registerCron();
+  await getService('weeklyMetrics').registerCron();
+  await getService('metrics').sendUploadPluginMetrics();
```

Comment:
We should not await telemetry calls

Reply:
👌 do yoou think I should remove the async in here?
I feel it could be safer to not await the telemetry call, but await this one just in case there is any error.

Reply:
I have added a change where I do not await the telemetry call itself, let me know what you think